README.md

```diff
@@ -38,3 +50,32 @@ Use `-h` to see options for specifying different types of input and output.
 
 You can also use fontmake to run intermediate steps in the build process, via
 methods of the `fontmake.font_project.FontProject` class.
+
+### Updating 
+
+To update the program the process is similar to initial installation, but there are a few minor differences. 
+
+Enter the virtual environment created for initial installation:
+
+```bash
+source env/bin/activate  # activate the environment (run `deactivate` to exit)
+```
+
+Enter this repo, and pull this repo:
+
+```bash
+cd fontmake;
+git clone git@github.com:googlei18n/fontmake.git;
+```
+
+Reinstall fontmake's dependencies to their latest versions:
+
+```bash
+pip install -r requirements.txt
+```
+
+Reinstall fontmake itself:
+
+```bash
+pip install -e .  # `-e` is for "editable" mode, only required for developers
```

Comment:
If you installed "editable" you don't need to re-run `pip install -e .`; `git pull` does the job.
If you installed without editable, then yes, you need to re-do `pip install .`